lib/reviews.js

```diff
@@ -57,15 +57,23 @@ function parseFields($) {
 		var title = reviewContent.find('span[class=review-title]').text().trim();
 		var text = filterReviewText(reviewContent.text().trim(), title.length);
 
+		var developerComment = $(this).next().first();
+		if (developerComment.attr("class") == "developer-reply")
+		{
```

Comment:
this curly brace should be on the previous line ;) (check the style files in the root dir)